### PR TITLE
Show errors for non x86 64 system descriptions for build and deploy

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 18e99a95186236334a0302da5f49387fde8f4597 into 7179ec894701e9d20ec9c290a6f9297b85c9f994 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into 7179ec894701e9d20ec9c290a6f9297b85c9f994 )

**Comment by cornelius** [comment]
Yay, pull request number 500 :fireworks:

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into 1e20fe81a10a2ed1662a4c16b427eef09df8a514 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into 3fd168162fd8397623ff865175442ee8758c99b9 )

File: lib/build_task.rb [hunk 1]
```diff
@@ -22,6 +22,12 @@ def build(system_description, output_path, options = {})
     LocalSystem.validate_existence_of_package("kiwi")
     LocalSystem.validate_existence_of_package("kiwi-desc-vmxboot")
 
+    if system_description.os.architecture != "x86_64"
```
Comment (thardeck) [inline lib/build_task.rb]: Shouldn't we extend the os class to also have information regarding the can_build architecture and not just the distribution?

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into c0ce4bd87fdabb0641991a694d820d142bfd60bb )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into 352a3609e43d25bf21f5879fbe55a5203270543a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 65940dfec11d862b4039440dfa1b17ff40e0d0d5 into 1039244745d666eb0aa31ebdcf8fd9e5c5cc6936 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 0c7ab47be5f2df81a0ac4c1b466e7be99eca2a27 into 1039244745d666eb0aa31ebdcf8fd9e5c5cc6936 )

**Pull request closed without merging**